web/fido_app/templates/profile.html

```diff
@@ -24,6 +24,8 @@
 <canvas width="400" height="400" id="snake-game" class="card"></canvas>
 
 <footer>
+  <a id="add-credential" class="btn btn-outline-primary btn-block profile-action {% if current_user.password_hash %}disabled{% endif %}" href="/add-password">Add Password</a>
+  <a id="add-biometric" class="btn btn-outline-primary btn-block profile-action {% if current_user.credential_id %}disabled{% endif %}" href="/webauthn/registration/add-start">Add Biometric</a>
```

Comment:
Rather than conditionally disabling these, I think it's better to conditionally render them. Something more like:
```jinja
{% if current_user.password_hash %}
  <a id="add-credential" class="btn btn-outline-primary btn-block profile-action" href="/add-password">Add Password</a>
{% endif %}
```

If we go this route though, the user doesn't know which credentials they have. Do you have any ideas to fix that issue?

Reply:
I actually approached it that way at first and rendered only one button for the credential the user doesn't have. I changed my mind for the reason you mentioned: it seemed like a good idea to make it explicit which credential the user had and didn't have.

Maybe we can keep the two buttons on screen together, but, say, if the user already has a password, we change the text on the disabled "Add Password" button to "Password Already Registered" or something like that?